src/menus/menus.controller.ts

```diff
@@ -106,4 +110,17 @@ export class MenusController {
   ) {
     await this.menuservice.deleteManyMenus(ids);
   }
+
+  @ApiResponse({
+    status: HttpStatus.OK,
+    description: 'The menu has been successfully published.',
+  })
+  @ApiResponse({
+    status: HttpStatus.NOT_FOUND,
+    description: 'No menu found',
+  })
+  @Patch(':id')
```

Comment:
Should be ':id/publish'

Reply:
Also don't forget to update the endpoint in tests after resolving this

Reply:
Done